### DISCUSSION TITLE
[Idea] Ability to add tags to extra media

### DISCUSSION THREAD

**Discussion by Cr1598** [sircharlo/meeting-media-manager#3373] [Ideas] [answered]
### What is your question?

In which directory can I find the images and videos that appear automatically sorted in the application?

### Is your question related to a problem?

In previous versions you could find the containing folder under "documents".

### Confirmations

- [X] I confirm that I have read the documentation website.
- [X] I confirm that I have searched through the existing discussions to make sure my question is not a duplicate.

**Comment by e-Feitosa** [comment]
In the current version you have the option to configure a monitored folder, maybe it will meet what you are looking for. The current M3 has changed the way it stores downloaded files. The new feature allows you to create and monitor a folder for example in Google Drive/ Dropbox etc. I hope it helped.

[](https://github.com/sircharlo/meeting-media-manager/discussions/2637)

**Comment by mtdvlpr** [comment]
The current version does not put all media, sorted, in a specific folder. Media is retrieved from various (cache) locations and sorting is saved somewhere else. Like mentioned already, we do have a feature that allows you to configure a folder with media that is automatically added to the media list.

**Reply by Cr1598** [reply]
I understand. I I wish that when you import an image, for example for the Watchtower study, to have the icon indicating it should be shared in paragraph, for example: ¶ 6.

**Reply by e-Feitosa** [reply]
@Cr1598, This only occurs when downloading media using your own M3. Which language is M3 using? But I understood what you mentioned, normally for the study of the W and Biblical Study of Congregation there is no need to include images. Additional media is usually for public talks.
![image](https://github.com/user-attachments/assets/c468d193-7a7b-4bcc-bf98-11ec397a018e)

**Reply by Cr1598** [reply]
I use the app in Spanish. I asked this because in the previous version I could place that character on the image I wanted. Is this no longer possible?

**Reply by sircharlo** [reply]
> I use the app in Spanish. I asked this because in the previous version I could place that character on the image I wanted. Is this no longer possible?

Do you mean manually indicating that this is a song or for a paragraph, and adding the numbers?

**Reply by Cr1598** [reply]
I need this because I would like to specify with this "¶" character that a certain image should be used in a certain paragraph.

**Reply by sircharlo** [reply]
I can add this as an "advanced" option hidden behind the right-click menu. Stay tuned.

**Reply by Cr1598** [reply]
That would be great! I'll be keeping an eye out! Thank you very much!

**Reply by sircharlo** [reply, marked as answer]
> That would be great! I'll be keeping an eye out! Thank you very much!

Pleasure! Added in https://github.com/sircharlo/meeting-media-manager/commit/d6183215eb9aad1921a7e1d70c8f93e422d146dd; will be released in the latest update. It's in the right-click menu for media items:

![image](https://github.com/user-attachments/assets/fe21e5a0-d36f-445b-a803-48ccdffb0041)
![image](https://github.com/user-attachments/assets/bcb41a9f-dbb1-43ce-b757-a1ef5166c295)

**Reply by Cr1598** [reply]
Wow! Awesome! Thank you very much! It will be very useful.
